frontend/views/containers/chatroom/MessageNotification.vue

```diff
@@ -57,7 +57,7 @@ export default ({
       const displayName = this.userDisplayName(username)
 
       const notificationTemplates = {
-        onGroupDM: {
+        onDirectMessage: {
```

Comment:
This is not using at the moment since we don't make notification messages inside the direct messages.
Even though it's not needed now, I didn't remove this because we could use this in the future.